packages/orchestration/test/facade.test.ts

```diff
@@ -127,4 +129,101 @@ test.serial('faulty chain info', async t => {
   });
 });
 
+test.serial('asset / denom info', async t => {
+  const { facadeServices, commonPrivateArgs } = await commonSetup(t);
+
+  // XXX relax again
```

Comment:
what failed without this? I wonder if we're better off addressing it. I could resume ,
- https://github.com/Agoric/agoric-sdk/pull/9677

Reply:
I was just following the pattern of other tests in this file.

If I take it out, I get

```
  test/facade.test.ts:137

   136:                                           
   137:   const { zcf } = await setupZCFTest();   
   138:   const zone = provideDurableZone('test');

  Rejected promise returned by test. Reason:

  Error {
    message: 'value is not durable: {"bundleCap":"[Alleged: FakeBundleCapHandle]","bundleID":"b1-contract"} at slot 0 of "#{\\"bundleCap\\":\\"$0.Alleged: FakeBundleCapHandle\\",\\"bundleID\\":\\"b1-contract\\"}"',
  }
```

Reply:
fair. When 9677 is ready I'll just go look for reincarnations that can be removed